### DISCUSSION TITLE
Sites using Remark42

### DISCUSSION THREAD

**Discussion by akellbl4** [umputun/remark42#1044] [Show and tell]
If you're using Remark42 at your company or on your site you can share the URL here.

**Comment by kostimoshenko** [comment]
I use it at my blog [konstantin.ist](https://konstantin.ist/) and I like it. Thank you guys for developing it further

**Comment by uPagge** [comment]
I run my blog about programming and use Remark42
Site: https://struchkov.dev/blog

**Reply by thunder-spb** [reply]
I'm using on my blog: https://iam.thunder.spb.ru/

**Comment by rwstorer** [comment]
The Remark42 project rocks! I added it to my Hugo site this weekend. It took me some time to get it working because I didn't realize SELinux was "getting in the way." I didn't disable SELinux. But, I did tweak the httpd_can_network_relay --> on. As soon as I did that, my 502 "bad gateway" went away, and it started working. I haven't finished setting up FaceBook OAuth (and a few others) and adding all the comment shortcodes to the site. I will soon.

Thanks again!

[https://rayramble.com](https://rayramble.com)

**Comment by vjt** [comment]
I use it on my blog https://sindro.me and I am thankful to you all for putting together this good piece of software!

**Comment by HyperTesto** [comment]
After trying out at least 5 comment engines on my blog (https://www.hypertesto.me), I landed on Remark42.
This is exactly what i was looking for both on functionalities and privacy standpoints!

**Comment by kaygdotorg** [comment]
Started using remark42 earlier this year and has been great: https://kayg.org/

**Comment by rbauduin** [comment]
I just deployed it on my blog and wrote about it at https://www.yvesdennels.com/posts/remark42-commenting/

**Comment by KaDeaT** [comment]
I added the remark42 system to my site: https://youngcoder.ru/lessons/1/
It's so beautiful and convenient!

**Comment by jpjonte** [comment]
Just added remark42 to my blog: https://jpj.dev/

Love it! It fulfills all my requirements: Self hosted, simple, easy to integrate, helm chart. ✅

**Comment by morikoff** [comment]
Thank you to the creator and all the developers!
This week, I installed Remark42 for my blog and wrote a post about it.
[https://morikoff.ru/replacing-disqus-with-remark42.html](https://morikoff.ru/replacing-disqus-with-remark42.html)
I recommend it to everyone who has **NOT** done so yet.

**Comment by geerlingguy** [comment]
My site: https://www.jeffgeerling.com

Just finished migrating my Drupal website to Hugo, and almost all my comments made it over to Remark42 using a [custom python migration script](https://github.com/geerlingguy/jeffgeerling-com/blob/master/drupal-export/drupal_to_remark42.py) — I have an issue documenting the entire migration process here: https://github.com/geerlingguy/jeffgeerling-com/issues/167
